exo/inference/tinygrad/inference.py

```
@@ -41,8 +41,19 @@
 
 def build_transformer(model_path: Path, shard: Shard, model_size="8B", device=None):
   # build model
+  try:
+    with open(model_path/"config.json", "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)